tpmutil/tbs/tbs_windows.go

```diff
@@ -250,3 +264,26 @@ func (context Context) GetTCGLog(logBuffer []byte) (uint32, error) {
 	)
 	return logBufferLen, getError(result)
 }
+
+// GetOwnerAuth Retrieves the owner authorization of the TPM if the information is available in the local registry.
+// If ownerAuthBuffer is nil, the actual size of the TPM ownerAuth is returned.
```

Comment:
Why not allocate the slice from within this function and return ([]byte, error)?